dts/bindings/mdio/mdio-controller.yaml

```diff
@@ -22,3 +22,10 @@ properties:
       When present, the SMA suppresses the 32-bit preamble and transmits
       MDIO frames with only 1 preamble bit. By default, the MDIO frame
       always has 32 bits of preamble as defined in the IEEE 802.3 specs.
+
+  clock-frequency:
+    type: int
+    description: |
+      Some MDIO controllers have the ability to configure the MDC frequency.
+      If present, this property may be used to specify the MDC frequency based
+      on what the PHYs connected to the mdio bus can support.
```

Comment:
how about having the default here rather than in the code? Also I think that the default should be mentioned in the description